cassandra/metadata.py

```diff
@@ -2626,28 +2625,24 @@ def get_table(self, keyspaces, keyspace, table):
         indexes_query = QueryMessage(
             query=maybe_add_timeout_to_query(self._SELECT_INDEXES + where_clause, self.metadata_request_timeout),
             consistency_level=cl, fetch_size=fetch_size)
-        triggers_query = QueryMessage(
-            query=maybe_add_timeout_to_query(self._SELECT_TRIGGERS + where_clause, self.metadata_request_timeout),
-            consistency_level=cl, fetch_size=fetch_size)
 
         # in protocol v4 we don't know if this event is a view or a table, so we look for both
         where_clause = bind_params(" WHERE keyspace_name = %s AND view_name = %s", (keyspace, table), _encoder)
         view_query = QueryMessage(
             query=maybe_add_timeout_to_query(self._SELECT_VIEWS + where_clause, self.metadata_request_timeout),
             consistency_level=cl, fetch_size=fetch_size)
         ((cf_success, cf_result), (col_success, col_result),
-         (indexes_sucess, indexes_result), (triggers_success, triggers_result),
+         (indexes_sucess, indexes_result),
          (view_success, view_result)) = (
              self.connection.wait_for_responses(
-                 cf_query, col_query, indexes_query, triggers_query,
+                 cf_query, col_query, indexes_query,
                  view_query, timeout=self.timeout, fail_on_error=False)
         )
         table_result = self._handle_results(cf_success, cf_result, query_msg=cf_query)
         col_result = self._handle_results(col_success, col_result, query_msg=col_query)
         if table_result:
             indexes_result = self._handle_results(indexes_sucess, indexes_result, query_msg=indexes_query)
-            triggers_result = self._handle_results(triggers_success, triggers_result, query_msg=triggers_query)
-            return self._build_table_metadata(table_result[0], col_result, triggers_result, indexes_result)
+            return self._build_table_metadata(table_result[0], col_result, None, indexes_result)
```

Comment:
Instead of passing None can you remove the argument from `_build_table_metadata`? Let's not introduce dead code.